package.json

```diff
@@ -27,18 +27,18 @@
     "lint:everything": "npm run lint && npm run test:types",
     "lint:fix": "standard --fix",
     "lint:standard": "standard --verbose | snazzy",
-    "test:mocha": "tap",
+    "test:unit": "c8 --statements 98 --branches 97 --functions 96 --lines 98 node --test",
```

Comment:
cannot do `c8 ---100` since there is not already 100% coverage being tracked here https://github.com/fastify/busboy/issues/126

Reply:
nice I didn't know we could separate them